src/apps/chifra/pkg/rpcClient/rpc_cmds_test.go

```diff
@@ -28,18 +26,18 @@ func TestFromRpcCounter(t *testing.T) {
 	}))
 	defer server.Close()
 
-	var result map[string]string
-	for i := 0; i < 20; i++ {
-		// TODO: Use rpc.Query
-		err := rpc.FromRpc(
-			server.URL,
-			&rpc.Payload{},
-			&result,
-		)
-		if err != nil {
-			t.Fatal(err)
-		}
-	}
+	// var result map[string]string
+	// for i := 0; i < 20; i++ {
+	// 	// TODO: Use rpc.Query
+	// 	err := rpc.fromRpc(
+	// 		server.URL,
+	// 		&rpc.Payload{},
+	// 		&result,
+	// 	)
+	// 	if err != nil {
+	// 		t.Fatal(err)
+	// 	}
+	// }
```

Comment:
Why are we commenting out this test?

Reply:
This is in the `rpcClient` package but it references `FromRpc` which I'd like to make internal only to the `rpc` package. I will move the test to the `rpc` package and uncomment it.